README.md

```diff
@@ -2,15 +2,52 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ome/omero-analysis-desktop/master?filepath=napari.ipynb)
 
 
-Run [OMERO clients](https://www.openmicroscopy.org/omero/downloads/) and [Napari](http://napari.org/) in a Linux desktop using Jupyter.
+Run [OMERO clients](https://www.openmicroscopy.org/omero/downloads/), [Napari](http://napari.org/) and [omero-napari](https://gitlab.com/openmicroscopy/incubator/omero-napari) in a Linux desktop using Jupyter.
```

Comment:
Before the incubator URL starts showing up in other resources, we should probably discuss a roll-out strategy.

Reply:
agree